zerver/openapi/zulip.yaml

```diff
@@ -17276,7 +17276,7 @@ components:
                 email:
                   type: string
                   description: |
-                    Email of the user.
+                    Zulip API email of the user.
```

Comment:
Again, clarified this as "Zulip API email", which is returned as a deprecated object in the emoji reaction events.